whispertext.py

```diff
@@ -0,0 +1,82 @@
+import pyaudio
+import threading
+import keyboard
+from faster_whisper import WhisperModel
+import io
+import numpy as np
+import soundfile as sf
+
+COMPUTE_TYPE = "float16"
+
+class Transcriber:
+    def __init__(self, model_name, device_type):
+        self.model = WhisperModel(model_name, device=device_type, compute_type=COMPUTE_TYPE)
+        self.recording = False
+        self.predicted_text = ""
+        self.frames = []
+        self.transcription_done = threading.Event()
+
+    def capture_audio(self):
+        audio = pyaudio.PyAudio()
+        stream = audio.open(format=pyaudio.paInt16, channels=1, rate=16000, input=True, frames_per_buffer=1024)
+
```

Comment:
Please make good allignment with no enpty lines, it makes it a bit harder to see where is the function.